staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -196,14 +196,20 @@ func (p *Patcher) Patch(current runtime.Object, modified []byte, source, namespa
 		if getErr != nil {
 			return nil, nil, getErr
 		}
-		patchBytes, patchObject, err = p.patchSimple(current, modified, source, namespace, name, errOut)
+		patchBytes, patchObject, err = p.patchSimple(current, modified, source, namespace, name, errOut, options)
 	}
 	if err != nil && (errors.IsConflict(err) || errors.IsInvalid(err)) && p.Force {
 		patchBytes, patchObject, err = p.deleteAndCreate(current, modified, namespace, name)
 	}
 	return patchBytes, patchObject, err
 }
 
+// Patch tries to patch an OpenAPI resource. On success, returns the merge patch as well
+// the final patched object. On failure, returns an error.
+func (p *Patcher) Patch(current runtime.Object, modified []byte, source, namespace, name string, errOut io.Writer) ([]byte, runtime.Object, error) {
```

Comment:
are there still any callers to this we need to update?

Reply:
> It seems like because there are so few callers it might be best to modify Patch to accept options, instead of having a separate PatchWithOptions. The diff caller can just past nil options like it currently does.

that seems better to me... then each caller gets to make a choice about what to pass in